src/main/java/org/ow2/proactive/catalog/repository/CatalogObjectRevisionRepositoryImpl.java

```diff
@@ -75,6 +75,16 @@ public List<CatalogObjectRevisionEntity> findDefaultCatalogObjectsOfKindListAndC
                  .collect(Collectors.toList());
     }
 
+    private String toRightSidePredicatePattern(String pattern) {
+        return pattern.endsWith("%") ? pattern.toLowerCase() : pattern.toLowerCase() + "%";
+    }
+
+    private String toBothSidesPredicatePattern(String pattern) {
+        String predicatePattern = toRightSidePredicatePattern(pattern);
+        predicatePattern = predicatePattern.startsWith("%") ? predicatePattern : "%" + predicatePattern;
```

Comment:
this does not seem to allow use input with "word%" as a percentage will always be added in the beginning.

Reply:
the idea is more "if the pattern contains % don't change it, otherwise add default patterns"